test/system/collaborator_accept_invitation.rb

```diff
@@ -0,0 +1,18 @@
+require "application_system_test_case"
+
+class CollaboratorAcceptInvitation < ApplicationSystemTestCase
+  test "create user account" do
+    collaborator = users(:collaborator1)
+    collaborator.invite!
+    token = collaborator.raw_invitation_token
+
+    visit accept_user_invitation_url(invitation_token: token)
+
+    fill_in "Name", with: collaborator.name
```

Comment:
Use `I18n.human_attribute_for(:name)` translation for the name of the fields

Reply:
We can't use `I18n.human_attribute_for(:name)` 
but works with 
`Client.human_attribute_name(:name)`

Reply:
ah, that's the helper I was referring to, yes use that